client/state/stored-cards/test/reducer.js

```diff
@@ -152,25 +152,6 @@ describe( 'items', () => {
 		} );
 	} );
 
-	test( 'should modify the stored card is_backup property correctly when is_backup is enabled', () => {
```

Comment:
Is there a reason for removing this test?

Reply:
No, I'm not sure why that got removed, but I've replaced it.